When I'am crosscompiling for Windows I will get the following error message:
```cpp
error(compilation): clang failed with stderr: In file included from /home/kimmo/duckdb/extension/parquet/parquet_writer.cpp:1:
In file included from /home/kimmo/duckdb/extension/parquet/include/parquet_writer.hpp:11:
In file included from /home/kimmo/duckdb/src/include/duckdb.hpp:11:
In file included from /home/kimmo/duckdb/src/include/duckdb/main/connection.hpp:14:
In file included from /home/kimmo/duckdb/src/include/duckdb/function/udf_function.hpp:11:
In file included from /home/kimmo/duckdb/src/include/duckdb/function/scalar_function.hpp:11:
In file included from /home/kimmo/duckdb/src/include/duckdb/common/vector_operations/binary_executor.hpp:12:
/home/kimmo/duckdb/src/include/duckdb/common/types/vector.hpp:72:13: error: attribute 'dllexport' cannot be applied to a deleted function
```

--- a/src/include/duckdb/catalog/catalog_search_path.hpp
+++ b/src/include/duckdb/catalog/catalog_search_path.hpp
@@ -22,7 +22,7 @@ class ClientContext;
 class CatalogSearchPath {
 public:
 	DUCKDB_API explicit CatalogSearchPath(ClientContext &client_p);
-//	DUCKDB_API CatalogSearchPath(const CatalogSearchPath &other) = delete;
+	CatalogSearchPath(const CatalogSearchPath &other) = delete;
 
 	DUCKDB_API void Set(const string &new_value, bool is_set_schema);
 	DUCKDB_API const vector<string> &Get();

--- a/src/include/duckdb/common/types/vector.hpp
+++ b/src/include/duckdb/common/types/vector.hpp
@@ -69,7 +69,7 @@ public:
 	*/
 	DUCKDB_API Vector(LogicalType type, bool create_data, bool zero_data, idx_t capacity = STANDARD_VECTOR_SIZE);
 	// implicit copying of Vectors is not allowed
-//	DUCKDB_API Vector(const Vector &) = delete;
+	Vector(const Vector &) = delete;
 	// but moving of vectors is allowed
 	DUCKDB_API Vector(Vector &&other) noexcept;
 

